### PR TITLE
chore: temporarily ignore failure core tests for hstore & simplify ci name

### PR DESCRIPTION
**Pull request by VGalaxies**
subtask of https://github.com/apache/incubator-hugegraph/issues/2420

### CODE REVIEW

File: hugegraph-server/pom.xml [hunk 1]
```diff
@@ -236,13 +236,11 @@
                 <groupId>io.grpc</groupId>
                 <artifactId>grpc-netty</artifactId>
                 <version>${grpc.version}</version>
-                <scope>provided</scope>
```
Comment (imbajin) [inline hugegraph-server/pom.xml:239]: any context with the modification?

File: .github/workflows/server-ci.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-name: "hugegraph-server-ci"
+name: "server CI"
```
Comment (imbajin) [inline .github/workflows/server-ci.yml:1]: - GraphServer CI (better?)
- Server CI
- graph server CI

Comment (VGalaxies) [reply .github/workflows/server-ci.yml:1]: `Graph Server CI` feels better, similar `Graph PD & Store & Hstore CI`

File: hugegraph-server/pom.xml (hunk 1 above)
Comment (VGalaxies) [reply hugegraph-server/pom.xml:239]: Dependencies marked as provided scope are available during compilation and testing, but they will not be included in the final WAR or JAR package.

If you use provided scope here, an error class not found will occur when running hstore with pd-client related code.

**Review by imbajin — DISMISSED** [review]

**Review by imbajin — APPROVED** [review]

**Review by JackyYangPassion — APPROVED** [review]

**Pull request merged**